docs/DeveloperGuide.md

```diff
@@ -38,19 +40,78 @@ Upon invoking the `Add` command, users are presented with a selection prompt fea
 input, typically in the form of a numerical identifier corresponding to a category within the enum class, facilitates
 the assignment of the appropriate enum object to the transaction's category attribute.
 
+### [Implemented] Process transaction
+
+#### Description
+
+This method adds a transaction to the list of transactions based on the necessary input details given by the user.
+
+#### Parameters
+
+1. String input: A string containing the user input, which should include the `NAME`, `AMOUNT`, `DATE` and `TYPE` of the
+   transaction.
+2. Account account: The account object associated with the transaction list.
+
+#### Design and Implementation
```

Comment:
Examples and expected outcomes could be used for clarity.

Reply:
Would it be clearer if we utilize more `markup format` for parameters?